projects/packages/forms/src/dashboard/components/InboxStatusToggle/index.js

```diff
@@ -0,0 +1,49 @@
+import {
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControl as ToggleGroupControl,
+	// eslint-disable-next-line @wordpress/no-unsafe-wp-apis
+	__experimentalToggleGroupControlOption as ToggleGroupControlOption,
+} from '@wordpress/components';
+import { useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import useStatusTabs from './useStatusTabs';
+import './style.scss';
+
+/**
+ * Renders the status toggle for the inbox view.
+ *
+ * @return {JSX.Element} The status toggle component.
+ */
+export default function InboxStatusToggle() {
+	const [ searchParams, setSearchParams ] = useSearchParams();
+	const status = searchParams.get( 'status' ) || 'inbox';
+
+	const statusTabs = useStatusTabs();
```

Comment:
Is there a reason for having this as a separate hook? It's only used here. Also if it was separate, it should be renamed, as it doesn't return `tabs`.

Reply:
It was just a lot of code focused on one purpose. I like it separated out, but if we'd prefer, we can keep the logic in the component file too.

It does return the array of tabs?

```
return [
    { label: getTabLabel( __( 'Inbox', 'jetpack-forms' ), totalItemsInbox ), value: 'inbox' },
    { label: getTabLabel( __( 'Spam', 'jetpack-forms' ), totalItemsSpam ), value: 'spam' },
    { label: getTabLabel( __( 'Trash', 'jetpack-forms' ), totalItemsTrash ), value: 'trash' },
];
```

Reply:
I don't feel too strongly on this, so I've moved the logic for getting the tabs into the component.